data_request_api/data_request_api/content/dreq_content.py

```diff
@@ -406,8 +403,6 @@ def retrieve(version="latest_stable", **kwargs):
                 json_export = _json_release
             elif kwargs["export"] == "raw":
                 json_export = _json_raw
-            else:
-                warnings.warn(f"Unknown export type '{kwargs['export']}'.")
```

Comment:
Removed this kind of warnings from `dreq_content` as this will now give `ValueError`s in the decorator.